augur/tasks/git/facade_tasks.py

```diff
@@ -255,7 +255,7 @@ def analyze_commits_in_parallel(repo_git, multithreaded: bool)-> None:
     facade_helper.log_activity('Debug',f"Commits missing from repo {repo_id}: {len(missing_commits)}")
 
     
-    if not len(missing_commits) or repo_id is None:
+    if missing_commits or repo_id is None:
```

Comment:
Did this flip the truthiness of this?

It sounds like not len would be true if len is 0, but the new version is true if missing_commits has nonzero length